### PR TITLE
README + help text updates

### PR DESCRIPTION
**Pull request by itmightbecody**
This PR expands the existing README to include additional information on usage, the structure of retrieved files, and troubleshooting steps. I've also suggested two help text updates for a typo and proper host URL structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,329 @@
-# skuid
-
-`skuid` is a command line application for interacting with Skuid metadata, on both Salesforce Platform and Skuid Platform.
-It provides ways to deploy and retrieve Skuid application metadata to and from either a Salesforce Platform org running the Skuid managed package or a Skuid Platform Site.
-
-## Motiviation
+# skuid CLI
```
Comment (ethanfrogers) [inline README.md]: Personally, I'd rather this stay `skuid` for consistency.

File: README.md [hunk 2]
```diff
@@ -1,134 +1,329 @@
...
+
+`skuid` is a command line application for retrieving and deploying Skuid objects (data and metadata) on both Skuid Platform sites and Salesforce orgs running the Skuid managed package.
+
+While Skuid is a cloud user experience platform, it can be helpful download an entire site's worth of pages to make small adjustments on multiple pages, to move them from a sandbox site to a production site, or to store them in a version control system (VCS). And while it's possible to save page XML and create page packs, moving entire swathes of metadata from site to site can prove challenging.
```
Comment (ethanfrogers) [inline README.md]: * Remove the "And" in "And while it's possible".
* replace "swathes" to "apps"

File: README.md [hunk 3]
```diff
@@ -1,134 +1,329 @@
...
+
+Enter the ``skuid`` command-line application. Using ``skuid`` you can easily pull—or download—your Skuid pages and push—upload—your Skuid metadata from one site to another using only a few commands.
```
Comment (ethanfrogers) [inline README.md]: only need one backtick for code formatting

File: README.md [hunk 4]
```diff
@@ -1,134 +1,329 @@
...
+* [Use Cases](#use-cases)
+	* [Version control](#version-control)
+	* [Sandboxes](#sandboxes)
+* [Troubleshooting](#troubleshooting)
+* [skuid vs skuid-grunt](#skuid-vs-skuid-grunt)
+* [Future](#future)
+
+## Prerequisites
+
+- A UNIX-based CLI, meaning you'll need to be working on a macOS or Linux machine.
```
Comment (ethanfrogers) [inline README.md]: replace "CLI" with "terminal". Examples: OSX - iTerm2 or Terminal app.

File: README.md [hunk 5]
```diff
@@ -1,134 +1,329 @@
...
 
-Our previous offering [`skuid-grunt`](https://bitbucket.org/skuid/skuid-grunt) provided this functionality as a Grunt plugin.
-It allowed users to configure Grunt tasks to perform pushes and pulls of Skuid Pages. This was great for projects already using NodeJS and Grunt
-but not so great if you really didn't want to require those dependencies. `skuid` solves that problem by producing a self-contained
-CLI to perform all the same operations.
+To manually install the application, follow these steps:
 
-## Examples (Salesforce Platform)
+1. Download [the latest release of the `skuid` CLI application binary.](https://github.com/skuid/skuid/releases)
+1. Rename the downloaded application binary file to ``skuid``.
```
Comment (ethanfrogers) [inline README.md]: example: `mv skuid_darwin_amd64 skuid`

File: README.md [hunk 6]
```diff
@@ -1,134 +1,329 @@
...
+1. Move the application to a folder in your `$PATH` variable, like `/usr/local/bin`, or add the application's folder to the PATH variable.
```
Comment (ethanfrogers) [inline README.md]: example `mv skuid /usr/local/bin/skuid`.

**Comment by ethanfrogers** [comment]
@itmightbecody Once we get past installation, do you think it might just be easier to link to our documentation rather to maintain it here _and_ in the docs site?

File: README.md (hunk 1 above)
Comment (itmightbecody) [reply README.md]: That makes sense. On the docs site I think we may still got with `skuid CLI` (or any other suggestions?) to better separate it in our table of contents. Just don't want one tiny topic to be confused with Skuid as a whole.

**Comment by itmightbecody** [comment]
@ethanfrogers Yeah I think that's a good idea. I'll address the comments here and then move that content into a docs PR. Does the rest of it look accurate?

**Pull request merged**